resolvers/webpack/test/files/webpack.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     { 'jquery': 'jQuery' },
     'bootstrap',
     function (context, request, callback) {
-      if (request === 'underscore') {
+      if (request === 'underscore' && context.indexOf('/') !== -1) {
```

Comment:
@Satyam: can you explain why this extra predicate? seems to be unnecessary on OSX, and I think it might be why the tests fail on Windows?